app/view_models/admin/needs_attention_auctions_view_model.rb

```diff
@@ -12,7 +12,7 @@ def drafts_partial
   end
 
   def drafts
-    @_drafts ||= Auction.unpublished.map do |auction|
+    @_drafts ||= AuctionQuery.new.unpublished.map do |auction|
```

Comment:
we moved the code that used to be in that `Scope` module within `AuctionQuery` into `AuctionScopes` so the behavior of `AuctionQuery` was clearer since before we were doing that confusing `define_method` stuff https://github.com/18F/micropurchase/commit/5d8cfaec763b8027303f171f85b8ffc36ef3c8ca#diff-8802c0b39ff625b976e3dc520209d322